app/actions/ModActionPanel/QuickAction.tsx

```diff
@@ -206,6 +207,7 @@ function Form(
   const deactivatedAt = getDeactivatedAt(
     repo ? { repo } : record ? { record } : {},
   )
+  const canManageChat = checkPermission('canManageChat')
```

Comment:
Also unrelated but this hides the `show messaging data` block from UI for external labelers.